lib/codelog/command/bump.rb

```diff
@@ -0,0 +1,48 @@
+module Codelog
+  module Command
+    class Bump
+      CHANGELOG_RELEASES_PATH = 'changelogs/releases/'.freeze
+      CHANGELOG_RELEASE_REGEXP = /^\d*\.{1}\d*\.{1}\d*$/
+      VALID_VERSION_TYPES = ['major', 'minor', 'patch'].freeze
+      INITIAL_RELEASE_VERSION = '0.0.0'.freeze
+
+      def self.run(version_type, release_date)
+        Codelog::Command::Bump.new.run version_type, release_date
+      end
+
+      def run(version_type, release_date)
+        unless VALID_VERSION_TYPES.include?(version_type)
+          abort(Codelog::Message::Error.invalid_version_type(version_type))
+        end
+
+        Codelog::Command::Release.run(next_version(version_type), release_date)
+      end
+
+      private
+
+      def next_version(version_type)
+        last_version = last_created_changelog.split('.').map(&:to_i)
```

Comment:
@Guilherme2112 what about making the `version_type` case insensitive, so we can use either `codelog bump major` or `codelog bump MAJOR`?

Reply:
@luisbevilacqua Added on 263fa23.